src/components/Whiteboard/ElementBehaviors/Resizable/ResizeHandle.tsx

```diff
@@ -209,7 +252,7 @@ export function ResizeHandle({
       scale={scale}
     >
       <rect
-        data-testid={`resize-handle-${cursor}`}
+        data-testid={`resize-handle-${rest.handlePosition}`}
```

Comment:
please use destructuring to  create a `handlePosition` before: `const { handlePosition } = ...`